downstairs/src/region.rs

```diff
@@ -647,14 +652,14 @@ impl Extent {
         let metadb = match open_sqlite_connection(&path) {
             Err(e) => {
                 println!(
-                    "Error: {} No extent#{} db file found for {:?}",
-                    e, number, path
+                    "Error: Open of db file {:?} for extent#{} returned: {}",
```

Comment:
I changed this because I saw this error when lacking enough file descriptors, but the message confused me into thinking a file was missing.  The error coming back from `open_sqlite_connection` is not really helpful either here, but it's all we have.